src/pyubx2/ubxreader.py

```diff
@@ -257,6 +262,7 @@ def _parse_rtcm3(self, hdr: bytes) -> tuple:
         Parse any RTCM3 data in the stream (using pyrtcm library).
 
         :param bytes hdr: first 2 bytes of RTCM3 header
+        :param bool parsing: a boolean flag to determine if the stream needs to be parsed

```

Comment:
can remove this docstring entry as it no longer applies

Reply:
Removed